src/frontend/src/flows/faq/index.ts

```diff
@@ -0,0 +1,70 @@
+import { html, render } from "lit-html";
+import { compatibilityChart } from "../../components/compatibilityChart";
+
+import { questions } from "./questions";
+
+// re-export for ease of use
+export { questions } from "./questions";
+
+// The FAQ page
+const pageContent = html`
+  <style>
+    /* briefly flash the question when redirected to a particular question */
+    @keyframes example {
```

Comment:
Let's call this animation something like `flash-question`?

Reply:
oh yeah

Reply:
Done!